src/election_notification_works/polling_places.clj

```
@@ -8,14 +8,17 @@
   and returns the polling places from Google's Civic Info API
   or an empty vector if none are found."
   [address & [google-election-id]]
-  (let [addr-str (str/join " " (map address [:street
-                                             :street-2
-                                             :city
-                                             :state
-                                             :zip]))
-        voter-info-partial (partial gc/voter-info (config [:google-api-key])
-                                    addr-str)
-        voter-info (if google-election-id
-                     (voter-info-partial google-election-id)
-                     (voter-info-partial))]
-    (:pollingLocations voter-info)))
+  (let [api-key (config [:google-api-key] :disabled)]
+    (if (= :disabled api-key)
```

We started with that, but the problem that led us to this is that returning `nil` as the default value here is indistinguishable from the env var not being set and using `#resource-config/env` in the `config.edn` (which we are). That leads to silent failures if you forget to set the env var.